### PR TITLE
add windows to supported, html tweak

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: octokit_utils.rb [hunk 1]
```diff
@@ -37,15 +38,15 @@ class OctokitUtils
     'puppet_authorization',
     'reboot',
     'registry',
-    'satellite',
+    'satellite_pe_tools',
```
Comment (tphoney) [inline octokit_utils.rb:41]: @wilson208  is this the right ?

Comment (wilson208) [reply octokit_utils.rb:41]: @tphoney Looking at the regex on line 51, looks like it requires the full name so 👍

File: release_planning.rb [hunk 2]
```diff
@@ -16,6 +16,11 @@
   opts.on('-t', '--oauth-token TOKEN', 'OAuth token. Required.') { |v| options[:oauth] = v }
   opts.on('-v', '--verbose', 'More output') { options[:verbose] = true }
   opts.on('-o', '--output', 'Creates html output') { options[:output] = true }
+
+	opts.on('--puppetlabs-supported', 'Select only Puppet Labs\' supported modules') {
```
Comment (DavidS) [inline release_planning.rb]: indentation here is screwed up

**Review by DavidS — APPROVED** [review]

**Comment by tphoney** [comment]
@DavidS fixed. sierra messed my .vimrc

**Pull request merged**